scripts/icons/convert_and_pack.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+if ! [ -x "$(command -v ./go-png2ico)" ]; then
+    echo "./go-png2ico not found"
+    echo "download the executable from https://github.com/J-Siu/go-png2ico"
+    echo "and drop it in this folder"
+    exit 1
+fi
+
+if ! [ -x "$(command -v optipng)" ]; then
+    echo "optipng executable not found"
+    exit 1
+fi
+
+if ! [ -x "$(command -v inkscape)" ]; then
+    echo "inkscape executable not found"
+    exit 1
+fi

```

Comment:
Can you add a comment about how to install these, similar to above? Or just a link to the source code's website or source code itself.

Reply:
Unlike the first one, these two projects are well-known. Can use scoop, can use chocolatey, can use winget.
Feel free to edit the script any way you want it, but as far as I'm concerted it's complete.

Reply:
The only thing though, since we're effectively committing half mb of binary data, might as well use the best png optimizer around. Will update the PR shortly.

Reply:
Done, saved around 20Kb total.

Reply:
I agree that inkscape is well know enough, never heard of the others.

This should probably be incorporated into CI to actually enforce it, but I guess that can be later.